demo-app/build.gradle.kts

```diff
@@ -68,7 +68,9 @@ android {
 
     buildTypes {
         release {
-            isMinifyEnabled = false
+            isMinifyEnabled = true
+            isShrinkResources = true
+            signingConfig = signingConfigs.findByName("debug")
```

Comment:
Not sure we want it like that, but it's nice for testing and we don't use it for anything else right now.